src/renderer/components/ft-icon-button/ft-icon-button.scss

```diff
@@ -97,7 +97,11 @@
     }
   }
 
-  &.favorite, &.favorite:hover, &.favorite:focus-visible {
+  &.favorite, &.favorite:hover, &.favorite:focus-visible, &.favorite:not(.disabled) {
+    &:not(.disabled) {
```

Comment:
Did you mean to add `:not(disabled)` twice?

Reply:
I did not lol, thank you